account_loan/model/account_loan.py

```diff
@@ -0,0 +1,475 @@
+# Copyright 2018 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT as DF
+
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+import logging
+import math
+
+_logger = logging.getLogger(__name__)
+try:
+    import numpy
+except (ImportError, IOError) as err:
+    _logger.error(err)
+
+
+class AccountLoan(models.Model):
+    _name = 'account.loan'
+    _description = 'Loan'
+    _inherit = ['mail.thread', 'mail.activity.mixin']
+
+    def _default_company(self):
+        force_company = self._context.get('force_company')
+        if not force_company:
+            return self.env.user.company_id.id
+        return force_company
+
+    name = fields.Char(
```

Comment:
Please add `copy=False`. Otherwise you get the error `Loan name must be unique` when copying an existing loan